src/ReinforcementLearningCore/src/core/stop_conditions.jl

```diff
@@ -1,8 +1,30 @@
 export StopAfterStep,
-    StopAfterEpisode, StopWhenDone, StopSignal, StopAfterNoImprovement, StopAfterNSeconds
+    StopAfterEpisode, StopWhenDone, StopSignal, StopAfterNoImprovement, StopAfterNSeconds, ComposedStopCondition
 
 import ProgressMeter
 
+
+#####
+# ComposedStopCondition
+#####
+
+"""
+    ComposedStopCondition(stop_conditions...; reducer = any)
+
+The result of `stop_conditions` is reduced by `reducer`.
+"""
```

Comment:
```suggestion
The result of `stop_conditions` is reduced by `reducer`. The default `reducer` is the `any` function, which means that the condition is true when any one of the `stop_conditions...` is true. Can be replaced by any function returning a boolean. For example `reducer = x->sum(x) >= 2` will require at least two of the conditions to be true.
"""
```
I added an extra example for the sake of clarity.